components/frontend-react/src/containers/LoginComponent.js

```diff
@@ -1,19 +1,49 @@
 import React from 'react';
 import Grid from '@material-ui/core/Grid';
+import axios from 'axios';
 
 import Appbar from '../components/Appbar';
 import BassaIntroBox from '../components/BassaIntroBox';
 import UserSignup from '../components/UserSignup';
 
 const LoginComponent = () => {
-  
-  const handleClickSubmit = (e) => {
-    e.preventDefault()
+
+  const handleClickSubmit = () => {
     console.log('Handling button submit click')
   }
+
+  const handleClickLogin = (username, pass) => {
+    //making axios POST call
+    let formData = new FormData();
+    formData.set("user_name", username);
+    formData.set("password", pass);
+    axios({
+      method: 'post',
+      url: `${process.env.REACT_APP_API_URL}/api/login`,
+      data: formData
+      },
+      {
+        headers: {'Content-Type': 'multipart/form-data' }
+      })
+    .then(res => {
+      makeGetRequest(res.headers.token);
+    })
+    .then(err => console.log(err));
+  }
+
+  const makeGetRequest = (token) => {
+    //Testing a GET request
```

Comment:
What is the purpose of this comment? If it is not necessary remove it?

Reply:
@JaDogg I will remove this function in the next PR. The function was created just to test whether GET requests are being made successfully.